qa/suites/rados/objectstore/backends/alloc-hint.yaml

```diff
@@ -11,7 +11,7 @@ overrides:
     conf:
       osd:
         filestore xfs extsize: true
-        osd objectstore: filestore
+        osd objectstore: bluestore
         osd op queue: wpq
```

Comment:
I think this file should just be removed.

Reply:
That means I can remove also test_alloc_hint.sh ?